exchanges/coinbene/coinbene_wrapper.go

```diff
@@ -0,0 +1,376 @@
+package coinbene
+
+import (
+	"fmt"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/orderbook"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/ticker"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/websocket/wshandler"
+	log "github.com/thrasher-corp/gocryptotrader/logger"
+)
+
+// Start starts the Coinbene go routine
+func (c *Coinbene) Start(wg *sync.WaitGroup) {
+	wg.Add(1)
+	go func() {
+		c.Run()
+		wg.Done()
+	}()
+}
+
+// Run implements the Coinbene wrapper
+func (c *Coinbene) Run() {
+	if c.Verbose {
+		log.Debugf("%s Websocket: %s. (url: %s).\n", c.GetName(), common.IsEnabled(c.Websocket.IsEnabled()), c.Websocket.GetWebsocketURL())
+		log.Debugf("%s polling delay: %ds.\n", c.GetName(), c.RESTPollingDelay)
+		log.Debugf("%s %d currencies enabled: %s.\n", c.GetName(), len(c.EnabledPairs), c.EnabledPairs)
+	}
+	exchangeCurrencies, err := c.GetAllPairs()
+	if err != nil {
+		log.Errorf("%s Failed to get available symbols.\n", c.GetName())
+	} else {
+		var newExchangeCurrencies currency.Pairs
+		for p := range exchangeCurrencies.Data {
+			newExchangeCurrencies = append(newExchangeCurrencies,
+				currency.NewPairFromString(exchangeCurrencies.Data[p].Symbol))
+		}
+		err = c.UpdateCurrencies(newExchangeCurrencies, false, true)
+		if err != nil {
+			log.Errorf("%s Failed to update available currencies %s.\n", c.GetName(), err)
+		}
+	}
+
+}
+
+// UpdateTicker updates and returns the ticker for a currency pair
+func (c *Coinbene) UpdateTicker(p currency.Pair, assetType string) (ticker.Price, error) {
+	var resp ticker.Price
+	var tempResp TickerResponse
+	var err error
+	allPairs := c.GetEnabledCurrencies()
+	for x := range allPairs {
+		tempResp, err = c.FetchTicker(exchange.FormatExchangeCurrency(c.Name, allPairs[x]).String())
+		if err != nil {
+			return resp, err
+		}
+		resp.Pair = allPairs[x]
+		resp.Last = tempResp.TickerData.LatestPrice
+		resp.High = tempResp.TickerData.DailyHigh
+		resp.Low = tempResp.TickerData.DailyLow
+		resp.Bid = tempResp.TickerData.BestBid
+		resp.Ask = tempResp.TickerData.BestAsk
+		resp.Volume = tempResp.TickerData.DailyVol
+		resp.LastUpdated = time.Now()
+		err1 := ticker.ProcessTicker(c.Name, &resp, assetType)
+		if err1 != nil {
+			return ticker.Price{}, err1
```

Comment:
return resp, err1

Reply:
Can also just reuse err here, but don't use the shorthand operator